root_links:
  - lbr4_0_link

tip_links:
  - lbr4_7_link

torque_limits:
  - 140.0 # A1 (J1) max: 176
  - 140.0 # A2 (J2) max: 176
  - 80.0  # E1 (J3) max: 100
  - 80.0  # A3 (J4) max: 100
  - 80.0  # A4 (J5) max: 100
  - 30.0  # A5 (J6) max: 38
  - 30.0  # A6 (J7) max: 38
